data/platforms.xml

```diff
@@ -37,8 +37,7 @@
 	<!-- ###################### BEGIN ALS Platforms ###################### -->
 	<platform id="copter_linearpath" name="Quadrocopter UAV" type="linearpath">
 		<scannerMount x="0" y="0" z="0.2">
-			<rot axis="x" angle_deg="180" />
-			<rot axis="z" angle_deg="175" />
+			<rot axis="x" angle_deg="175" />
```

Comment:
the beam of the scanner is defined as
```xml
<beamOrigin x="0" y="0.085" z="0.06">			
		<rot axis="x" angle_deg="90" />			
		<rot axis="z" angle_deg="90" />			
</beamOrigin>
```
To my understanding, this should turn the scanner from scanning "forward, up-to-down" to "**UPwards**, left-to-right" (cf. https://github.com/3dgeo-heidelberg/helios/wiki/Scanners#scanner-system-definitions).

The subsequent rotation about the x-axis by 175 deg would then point it downwards (5deg off-nadir), left-to-right. 

I guess my question is why are we pointing it upwards first and then downwards, instead of doing 
in the Scanner:

```xml	
		<rot axis="x" angle_deg="-90" />			
		<rot axis="z" angle_deg="90" />		
```

in the Platform:
```xml
<rot axis="x" angle_deg="-5" />
```


https://github.com/3dgeo-heidelberg/helios/wiki/Platforms#rotations

Reply:
In this demo both platforms are compared. Both surveys are presented but the focus is on the ULS survey, that is why the platform and the scanner are only presented for the ULS survey. At the end both results are compared. Or did I misunderstand the problem?

I corrected the spelling mistake and adapted the thumbnail so far.